internal/services/cockpit/alert_manager.go

```diff
@@ -0,0 +1,250 @@
+package cockpit
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/cockpit/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/locality/regional"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/services/account"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/types"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/verify"
+)
+
+func ResourceCockpitAlertManager() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: ResourceCockpitAlertManagerCreate,
+		ReadContext:   ResourceCockpitAlertManagerRead,
+		UpdateContext: ResourceCockpitAlertManagerUpdate,
+		DeleteContext: ResourceCockpitAlertManagerDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": account.ProjectIDSchema(),
+			"enable_managed_alerts": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Enable or disable the alert manager",
+			},
+			"emails": {
```

Comment:
The [ContactPoint](https://github.com/scaleway/scaleway-sdk-go/blob/master/api/cockpit/v1/cockpit_sdk.go#L388C1-L396C2) structure, which contains emails, has certainly been designed so that other means of contact can be added in the future, so perhaps it would be more judicious and future-proof to manage the ContactPoint field directly

```suggestion
			"contact_point": {
                                  Type:        schema.TypeList,
                                  Optional:    true,
                                  Description: "foo",
                                  Elem: &schema.Resource{
                                     Schema: map[string]*schema.Schema{
                                        "email": {
                                           Type:     schema.TypeString,
                                           Optional: true,
                                           ValidateFunc: verify.IsEmail(),
                                           Description: "foo",
                                        },
                                     },
                                  },
                              },
```